README.md

```diff
@@ -129,24 +138,31 @@ We recommend exploring the contents of this project and familiarizing yourself w
    Depending on your requirements, you can choose to deploy all CloudFormation templates or individual templates.
    
     ```
-    $ cdk deploy --all
+    $ cdk deploy --all (for SQS) or (for Firehose) $ cdk deploy --all --context delivery_type=firehose
```

Comment:
i recommend to split this sample command into multiple commands that users can directly copy execute for each scenario